framework/base/Model.php

```diff
@@ -566,6 +577,30 @@ public function getErrors($attribute = null)
     }
 
     /**
+     * Returns specific attribute related errors only (without common errors).
+     * @return array errors for all attributes
+     * @see getErrors()
+     * @since 2.0.11
+     */
+    public function getAttributeErrors()
+    {
+        $errors = $this->getErrors();
+        unset($errors[self::COMMON_ERRORS_KEY]);
+        return $errors;
+    }
+
+    /**
+     * Returns the common errors only
+     * @return array array of common error messages
+     * @see getErrors()
+     * @since 2.0.11
+     */
+    public function getCommonErrors()
```

Comment:
if this method is the counterpart of geAttributeErrors(), it would be logical to name it getNonAttributeErrors()